proto/v1/message.proto

```diff
@@ -54,3 +54,48 @@ message Sources {
     repeated string multiaddrs = 2;
   }
 }
+
+
+message BidBotEvent {
+  google.protobuf.Timestamp ts = 1;
```

Comment:
Adding this considering: 1) it may take longer than anticipated time for the event to arrive auctioneer; 2) the local time where bidbot runs may not be synchronized. Both are very unlikely, so not sure if it's worth adding this field for every message. 

Imagine a timeline of events for a bid showing up on the UI, for consistency we may prefer to show the timestamp the events being received by auctioneer. your thoughts @asutula ?

Reply:
> Imagine a timeline of events for a bid showing up on the UI, for consistency we may prefer to show the timestamp the events being received by auctioneer.

:100:

Reply:
I'd say it's better to over collect here so we have the options of displaying either or both of the timestamps (bidbot and auctioneer).